packages/url/src/index.js

```diff
@@ -34,6 +34,51 @@ export function addQueryArgs( url, args ) {
 	return baseUrl + '?' + stringify( { ...query, ...args } );
 }
 
+/**
+ * Returns a single query argument of the url
+ *
+ * @param  {string} url URL
+ * @param  {string} arg Query arg name
+ *
+ * @return {Array|string} Query arg value.
+ */
+export function getQueryArg( url, arg ) {
+	const queryStringIndex = url.indexOf( '?' );
+	const query = queryStringIndex !== -1 ? parse( url.substr( queryStringIndex + 1 ) ) : {};
+
+	return query[ arg ];
+}
+
+/**
+ * Determines whether the URL contains a given query arg.
+ *
+ * @param  {string}    url URL
```

Comment:
Per standards examples, we've avoided bothering to align the `@param` type with `@return`.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions